guides/source/action_cable_overview.md

```diff
@@ -242,9 +242,9 @@ WebNotificationsChannel.broadcast_to(
 ```
 
 The `WebNotificationsChannel.broadcast_to` call places a message in the current
-subscription adapter (by default `redis` for production and `async` for development and
-test environments)'s pubsub queue under a separate broadcasting name for each user.
-For a user with an ID of 1, the broadcasting name would be `web_notifications:1`.
+subscription adapter's pubsub queue under a separate broadcasting name for each user.
+The default pubsub queue for ActionCable is `redis` in production and `async` in development and
```

Comment:
Sorry, I didn't catch it the first time around but could you change `ActionCable` to `Action Cable` please ? The former is the Ruby constant, the latter is the library itself.